TLM/TLM/Resources/lang_fr.txt

```diff
@@ -175,60 +175,60 @@ default par défaut
 flow_ratio ratio de mouvement
 wait_ratio ratio d'attente
 After_min._time_has_elapsed_switch_to_next_step_if Après le temps minimum écoulé, passez à l'étape suivante si
-Adaptive_step_switching Commutation d'étape adaptive
+Adaptive_step_switching Commutation d'étape adaptative
 Dynamic_lane_section Sélection de voie dynamique
 Percentage_of_vehicles_performing_dynamic_lane_section Pourcentage de véhicules effectuant une sélection de voie dynamique
 Vehicle_restrictions_aggression Restrictions agressives des véhicules
 Strict Stricte
-Show_path-find_stats Afficher le chemin d'acès des stats
+Show_path-find_stats Afficher le chemin d'accès des stats
 Remove_this_vehicle Retirer ce véhicule
-Vehicles_follow_priority_rules_at_junctions_with_timed_traffic_lights Les véhicules suivent les règles de priorité aux carrefours avec des feux de circulations temporiés
-Enable_tutorial_messages Activier les messages du didactitiel
-TMPE_TUTORIAL_HEAD_MainMenu Gestionnaire de traffic : Édition Président
-TMPE_TUTORIAL_BODY_MainMenu Bienvenue à TM:PE !\n\nManuel d'utilisateur : http://www.viathinksoft.de/tmpe
+Vehicles_follow_priority_rules_at_junctions_with_timed_traffic_lights Les véhicules suivent les règles de priorité aux carrefours avec des feux de circulations chronométrés
+Enable_tutorial_messages Activer les messages du didactitiel
+TMPE_TUTORIAL_HEAD_MainMenu Gestionnaire de trafic : Édition Président
+TMPE_TUTORIAL_BODY_MainMenu Bienvenue sur TM:PE !\n\nManuel d'utilisateur : http://www.viathinksoft.de/tmpe
 TMPE_TUTORIAL_HEAD_JunctionRestrictionsTool Restrictions aux carrefours
-TMPE_TUTORIAL_BODY_JunctionRestrictionsTool Comment contrôler le comportement des véhicules et des piétons aux carrefours.\n\n1. Cliquer sur le carrefour que vous voulez gérer\n2. Cliquez sur l'icône appropriée pour basculer entre les restrictions.\n\nRestrictions disponibles :\n- Autoriser/Interdire le changement de voie pour que le véhicule continue tout droit\n- Autoriser/Interdire les demi-tours\n- Autoriser/Interdire aux véhicules d'entrer dans un carrefour bloqué\n- Autoriser/Interdire aux pietons de traverser la rue
-TMPE_TUTORIAL_HEAD_LaneArrowTool Flèche des voies de circulation
+TMPE_TUTORIAL_BODY_JunctionRestrictionsTool Comment contrôler le comportement des véhicules et des piétons aux carrefours.\n\n1. Cliquez sur le carrefour que vous voulez gérer\n2. Cliquez sur l'icône appropriée pour basculer entre les restrictions.\n\nRestrictions disponibles :\n- Autoriser/Interdire le changement de voie pour que le véhicule continue tout droit\n- Autoriser/Interdire les demi-tours\n- Autoriser/Interdire aux véhicules d'entrer dans un carrefour bloqué\n- Autoriser/Interdire aux pietons de traverser la rue
+TMPE_TUTORIAL_HEAD_LaneArrowTool Directions des voies de circulation
```

Comment:
I think `Flèche` might be better in this instance as the feature is specific to lane _arrows_ and thus needs disambiguation from the separate lane routing/connector tool. What do you think?

Reply:
I modified this to make it consistent with the rest of the file, where every time `Lane Arrow` is mentioned, it has been translated to `Direction` in fr.
For instance
`Change_lane_arrows` `Changer les directions de voies`
`TMPE_TUTORIAL_BODY_LaneArrowTool` `Restreindre l'ensemble des directions [...]`

The connector tool is currently litterally translated to `connecteur`.

Reply:
If someone in France says `Directions des voies de circulation` will people immediately think of lane arrows? If so, it's correct term, otherwise it would need changing.

Reply:
I'm going to propose `Flèches directionnelles` as this sounds to me as the most evident to refer to the painted arrows, and is more concise.

Reply:
Yes, that is good!